dask/dataframe/io/tests/test_parquet.py

```diff
@@ -577,7 +587,7 @@ def test_categorical(tmpdir, write_engine, read_engine):
     ddf2 = dd.read_parquet(tmp, categories=[], engine=read_engine)
 
     ddf2.loc[:1000].compute()
-    assert (df.x == ddf2.x).all()
+    assert (df.x == ddf2.x.compute()).all()
```

Comment:
I needed to get this test passing locally with the latest pandas (where the categorical groupby bug is fixed). But since this wasn't failing with older pandas (where this test isn't skipped), investigated this a bit more, and this might be another regression in pandas -> https://github.com/pandas-dev/pandas/issues/38946